src/main/java/org/broadinstitute/consent/http/resources/SupportResource.java

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.consent.http.resources;
+
+import com.google.api.client.http.HttpStatusCodes;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.inject.Inject;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.broadinstitute.consent.http.models.support.DuosTicket;
+import org.broadinstitute.consent.http.models.support.TicketFactory;
+import org.broadinstitute.consent.http.models.support.TicketFields;
+import org.broadinstitute.consent.http.service.SupportRequestService;
+import org.broadinstitute.consent.http.util.gson.GsonUtil;
+import org.owasp.fileio.FileValidator;
+import org.zendesk.client.v2.model.Request;
+
+@Path("support")
+public class SupportResource extends Resource {
+
+  private final SupportRequestService supportRequestService;
+  private final static TicketFactory ticketFactory = new TicketFactory();
+  private final static Gson gson = GsonUtil.getInstance();
+  private final static FileValidator validator = new FileValidator();
+
+  @Inject
+  public SupportResource(SupportRequestService supportRequestService) {
+    this.supportRequestService = supportRequestService;
+  }
+
+  @POST
+  @Path("request")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response postRequest(String body) {
+    try {
+      TicketFields ticketFields = gson.fromJson(body, TicketFields.class);
+      DuosTicket ticket = ticketFactory.createTicket(ticketFields);
+      logInfo("Support Request Ticket: " + ticket.toString());
```

Comment:
`toString()` is implicit in a string concatenation expression
```suggestion
      logInfo("Support Request Ticket: " + ticket);
```